test/known_issues/test-stdout-buffer-flush-on-exit.js

```diff
@@ -5,16 +5,23 @@ require('../common');
 const assert = require('assert');
 const execSync = require('child_process').execSync;
 
-const longLine = 'foo bar baz quux quuz aaa bbb ccc'.repeat(65536);
+const lineSeed = 'foo bar baz quux quuz aaa bbb ccc';
 
 if (process.argv[2] === 'child') {
+  const longLine = lineSeed.repeat(Math.pow(2, parseInt(process.argv[3], 10)));
   process.on('exit', () => {
     console.log(longLine);
   });
   process.exit();
 }
 
-const cmd = `${process.execPath} ${__filename} child`;
-const stdout = execSync(cmd).toString().trim();
+[16, 18, 20].forEach((exponent) => {
+  console.log(exponent);
```

Comment:
I accidentally left it in from debugging. I can add it to the assert message, perhaps.